pkg/processor/trigger/partitioned/v3io/partition.go

```diff
@@ -131,3 +136,29 @@ func (p *partition) getRecords(partitionPath string, location string, pollingInt
 	return response.Output.(*v3iohttp.GetRecordsOutput), nil
 
 }
+
+func (p *partition) waitPartitionAvailable(partitionPath string, pollingInterval time.Duration) error {
+	responseChannel := make(chan *v3iohttp.Response)
+
+	for {
+		request := &v3iohttp.ListBucketInput{
+			Path: p.v3ioTrigger.streamPath,
+		}
+		p.v3ioTrigger.container.ListBucket(request, context.TODO(), responseChannel)
```

Comment:
Unhandled error.

Reply:
Also, why not use sync API here?
```
response, err := p.v3ioTrigger.container.Sync.ListBucket(request)
```

Reply:
Also, looks like this code is missing a call to `response.Release()`

Reply:
underlying function doing the release for me